clouddrift/dataformat.py

```diff
@@ -377,3 +377,23 @@ def to_parquet(self, filename: str):
         """
         ak.to_parquet(self.to_awkward(), filename)
         return
+
+
+def unpack(ragged_array: np.ndarray, rowsize: np.ndarray[int]) -> list[np.ndarray]:
+    """Unpack a ragged array into a list of regular arrays.
+
+    Unpacking ragged_array as an np.ndarray is about 2 orders of magnitude
+    faster than unpacking it as an xr.DataArray, so unles you need a DataArray
```

Comment:
faster than unpacking a xr.DataArray ragged array, so unless you need a DataArray